bigchaindb/core.py

```diff
@@ -485,3 +485,25 @@ def get_unvoted_blocks(self):
             unvoted.pop(0)
 
         return unvoted
+
+    def block_election_status(self, block):
+        """
+        Tallies the votes on a block, and returns the status: valid, invalid, or undecided
+        """
+        n_voters = len(block['block']['voters'])
+
+        vote_list = [vote['vote']['is_block_valid']
+                     for vote in block['block']['votes']
+                     if self.consensus.verify_vote_signature(vote)]
+
+        # validate votes here
+
+        n_invalid_votes = vote_list.count(False)
+        n_valid_votes = vote_list.count(True)
+
+        if n_invalid_votes >= int(n_voters/2):
```

Comment:
Also, `int` is equivalent to `math.floor` for positive numbers.

In this scenario:
 - `n_voters` is odd, let's say `5`
 - `vote_list` is `[True, True, False, False]`
 - `n_invalid_votes` is `2`
 - `int(n_voters/2)` is `2`

the function would return `invalid`, is that what we want?

An alternative approach is to add a new variable, called `quorum` (so it's more explicit), that is `math.ceil(n_votes / 2)` or `round(n_votes / 2)` and then branch against it.